.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 110
```

Comment:
Looks like for black there is `line-length = 100`, should not those match?

Reply:
@pinkavaj my apologies, I forgot to delete the unused `.flake` file

Reply:
noo need for apology